src/server/config/schema.js

```diff
@@ -81,6 +81,10 @@ module.exports = () => Joi.object({
   })
   .default(),
 
+  ops: Joi.object({
```

Comment:
thoughts on nesting this in an object called `monitor` or similar instead of ops?  i see other monitor configs in the hapi docs that may end up here.  are there any other configs you see being in the ops object?